brands/amenity/bank.json

```diff
@@ -3459,6 +3468,17 @@
       "name": "Federal Bank"
     }
   },
+  "amenity/bank|Ficohsa": {
```

Comment:
For this one, I notice:
* It operates in countries other than Honduras. (see https://www.ficohsa.com/)
* `brand` and `name` tags should just be "Ficohsa"
* need to replace underscores with spaces in the `brand:wikipedia` tag
* remove the redundant matchName